src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -1334,3 +1334,306 @@ def test_WINNF_FT_S_IPR_7(self, config_filename):
+          ssl_key=domain_proxy_config['key'])
+      domain_proxy.registerCbsdsAndRequestGrants(
+          domain_proxy_config['registrationRequests'],
+          domain_proxy_config['grantRequests'],
+          conditional_registration_data=domain_proxy_config[
+              'conditionalRegistrationData'])
+      domain_proxies.append(domain_proxy)
+
+    logging.info('Steps 7 + 8: generate and pull FAD.')
+    sas_uut_fad = None
+    test_harness_fads = []
+    if num_peer_sases:
+      # Trigger, wait and download FAD records from SAS UUT and test harnesses.
+      ssl_cert = config['sasTestHarnessConfigs'][0]['serverCert']
+      ssl_key = config['sasTestHarnessConfigs'][0]['serverKey']
+      sas_uut_fad = getFullActivityDumpSasUut(
+          self._sas, self._sas_admin, ssl_cert=ssl_cert, ssl_key=ssl_key)
+      for test_harness in test_harnesses:
+        test_harness_fads.append(
+            getFullActivityDumpSasTestHarness(
+                test_harness.getSasTestHarnessInterface()))
+
+    # Trigger CPAS.
+    logging.info('Step 9: Triggering CPAS.')
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    logging.info('Step 10: (done below).')
+
+    # Inform the SAS UUT that the ESC-monitored DPA is activated.
+    if using_esc_dpa:
+      if dpa_config['frequencyRange']['lowFrequency'] < LOW_FREQUENCY_LIMIT_HZ:
+        logging.info(
+            'Skipping Step 11: an always-active DPA was specified, so there is '
+            'no need to activate.'
+        )
+      else:
+        dpa_activation_request = {
+            'dpaId': dpa_config['dpaId'],
+            'frequencyRange': dpa_config['frequencyRange']
+        }
+        logging.info('Step 11: activating DPA: %s', dpa_activation_request)
+        self._sas_admin.TriggerDpaActivation(dpa_activation_request)
+    else:
+      logging.info(
+          'Skipping Step 11: using portal-controlled DPA which is active by '
+          'default.'
+      )
+
+    logging.info('Step 12: wait + heartbeat.')
+    time.sleep(240)
+    for domain_proxy in domain_proxies:
+      domain_proxy.heartbeatForAllActiveGrants()
+    grant_info = data.getAuthorizedGrantsFromDomainProxies(domain_proxies)
+
+    logging.info('Step 10, 13, + CHECK: DPA aggregate interference check.')
+    logging.info('Checking DPA %s', dpa_config)
+    dpa_filename = config['dpaDatabaseConfig'][
+        'filePath'] if using_portal_dpa else None
+    dpa = dpa_mgr.BuildDpa(
+        dpa_config['dpaId'],
+        dpa_config['points_builder'],
+        portal_dpa_filename=dpa_filename,
+        use_updated_neighborhoods=True,
+        apply_clutter_and_network_loss=True)
+    low_freq_mhz = dpa_config['frequencyRange']['lowFrequency'] // ONE_MHZ
+    high_freq_mhz = dpa_config['frequencyRange']['highFrequency'] // ONE_MHZ
+    dpa.ResetFreqRange([(low_freq_mhz, high_freq_mhz)])
+    dpa.SetGrantsFromFad(sas_uut_fad, test_harness_fads)
+    dpa.ComputeMoveLists()
```

Comment:
Remove ComputeMoveLists, IPR.8 shouldn't have peer SAS so this is not needed.

Reply:
Done